doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/primitives/text/localization/TextLocalizationPage.swift

```diff
@@ -0,0 +1,36 @@
+//
```

Comment:
Is there any reason that we have empty comment lines at the beginning of the files?